galasa-ui/src/components/headers/ThemeSelector.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+'use client';
+
+import React, { useTransition } from "react";
+import { Dropdown } from "@carbon/react";
+import styles from "@/styles/Selector.module.css"; 
+import { useTheme } from "@/contexts/ThemeContext";
+import { useTranslations } from "next-intl";
+
+type ThemeType = "white" | "g100"; 
+
+const themes = [
+  { id: "white", text: "White", value: "white" },
+  { id: "g100", text: "Dark", value: "g100" },
+];
+
+export default function ThemeSelector() {
+
+  const { theme, setTheme } = useTheme();
+  const [isPending, startTransition] = useTransition();
+  const selectedItem = themes.find((t) => t.value === theme) || themes[0];
+  const translations = useTranslations('ThemeSelector');
+  
+  const handleThemeChange = ({ selectedItem }: { selectedItem: { id: string; text: string; value: string } }) => {
+    if (!selectedItem) return;
```

Comment:
Instead of an early return here, we could invert the condition to only continue with `if (selectedItem)`?